src/NLog/Internal/FileAppenders/CountingSingleProcessFileAppender.cs

```
@@ -83,8 +84,20 @@ public override void Close()
         {
             if (this.file != null)
             {
-                this.file.Close();
-                this.file = null;
+                InternalLogger.Trace("Closing '{0}'", FileName);
+
+                try
+                {
+                    this.file.Close();
+                }
+                catch (Exception ex)
+                {
+                    InternalLogger.Warn(ex, "Failed to close file: '{0}'", FileName);
```

Think we should mention we are ignoring the exception